packages/blade/src/components/Typography/BaseText/BaseText.tsx

```diff
@@ -27,6 +27,10 @@ export type BaseTextProps = {
   truncateAfterLines?: number;
   className?: string;
   children: React.ReactNode;
+  /**
+   * @platform native
+   */
+  accessibilityRole?: string;
```

Comment:
I thought we'll use the mapA11yProps function to do this on all components? Why are we doing this directly?